sdk/eventhub/Azure.Messaging.EventHubs/src/Core/MessagePropertyName.cs

```diff
@@ -12,8 +12,8 @@
     ///
     internal static class MessagePropertyName
     {
-        /// <summary>The date and time, in UTC, that a message was enqueued.</summary>
```

Comment:
Let's please leave the explicit UTC in the comments; it certainly doesn't hurt.   Honestly, outside of Microsoft, I haven't seen anyone really make use of `DateTimeOffSet` so the additional context will be helpful, I feel.

Reply:
Done.